src/components/SendFormModal/SendFormModal.tsx

```diff
@@ -1,150 +1,171 @@
-                                underline: { inline: 'u', exact: true }
-                            },
-                            placeholder: "Optional Note(s)...",
-                            height: '20rem',
-                            menubar: false,
-                            auto_focus: true,
-                            //statusbar: false, //Have to show this as part of the license agreement if we don't attribute elsewhere
-                            plugins: [
-                                'advlist autolink lists link charmap anchor',
-                                'searchreplace',
-                                'insertdatetime paste wordcount'
-                            ],
-                            toolbar: 'undo redo | bold italic underline | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent link'
-                        }}
-                        onEditorChange={(content) => setNoteText(content)}
-                    />
-                </div>
-            </Form>
-        </SubmittableModal>
-    );
-
-}
\ No newline at end of file
+  return (
+    <SubmittableModal
+      modalTitle={nextStageText(props.process)}
+      show={props.showModal}
+      submitDisabled={props.submitDisabled}
+      onShow={props.onShow}
+      handleClose={closeForm}
+      submit={submitForm}
+    >
+      <Form>
+        {props.process.CurrentStage === Stages.SBP_REVIEW && (
+          <Form.Group controlId="nextStage">
+            <Form.Label className="required">
+              <strong>Next Stage</strong>
+            </Form.Label>
+            <Form.Control
+              as="select"
+              value={nextStage}
+              onChange={(e) => setNextStage(e.target.value as Stages)}
+              className="mb-2"
+            >
+              <option>{Stages.SBA_PCR_REVIEW}</option>
+              <option>{Stages.CO_FINAL_REVIEW}</option>
+            </Form.Control>
+          </Form.Group>
+        )}
+        <Form.Group controlId="personSelect">
+          <Form.Label className="required">
+            <strong>
+              {nextStage === Stages.COMPLETED ? "Buyer:" : "Reviewer:"}
+            </strong>
+          </Form.Label>
+          <Form.Control
+            as={PeoplePicker}
+            defaultValue={assignee ? [assignee] : undefined}
+            updatePeople={(p: IPerson[]) => {
+              let persona = p[0];
+              setAssignee(persona ? new Person(persona) : undefined);
+            }}
+            required
+            isInvalid={submitAttempted && assignee === undefined}
+          />
+          {submitAttempted && assignee === undefined && (
+            <Form.Control.Feedback type="invalid">
+              You must provide a{" "}
+              {nextStage === Stages.CO_FINAL_REVIEW ? "Buyer" : "Reviewer"}
+            </Form.Control.Feedback>
+          )}
+        </Form.Group>
+        <div className="mt-2 mb-2">
+          <Form.Group controlId="formNotes">
```

Comment:
Not sure this is working to properly label the TinyMCE box, since formNotes doesn't exist in rendered HTML